web/web_handler.go

```diff
@@ -50,8 +50,6 @@ func (h *EventHandler) Handle(w http.ResponseWriter, r *http.Request) {
 		}
 
 		metrics.Mark("events.requests." + e.Type)
-		delay := time.Now().Unix() - e.Timestamp.Unix()
-		metrics.UpdateGauge("events.requests.delay.current", delay)
```

Comment:
Similar is pushed from event_handler.go:106,156 based on event after event parsing now.
Keep it for clarity ?